config/locales/en.yml

```diff
@@ -863,6 +864,9 @@ en:
         title: Workflow Selection
       new_automated_workflow_execution_modal:
         title: "%{workflow} parameters"
+      edit_dialog:
+        title: "Editing Automated Workflow Execution: %{workflow_name}"
+        submit: Save changes
```

Comment:
**nitpick (blocking):** I believe `submit` is unused here, if so can you remove it.

Reply:
Removed in 2fa5e21